src/test/java/core/basesyntax/DataConverterTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class DataConverterTest {
+
+    @Test
+    void testConvert_EmptyData() {
```

Comment:
please adhere to the suggested in the task description naming convention, third part of name it expected outcome